tetris.js

```diff
@@ -518,7 +518,64 @@ module.exports = {
 		let arrayTypeOfTetramino = ["L", "J", "T", "S", "Z", "I", "O"];
 		return arrayTypeOfTetramino[Math.floor(Math.random() * arrayTypeOfTetramino.length)];
 	},
-	initCoordinates: function (typeOfTetromino, fieldSize ) {
-
+	initCoordinates: function (typeOfTetromino, fieldSize) {
```

Comment:
Rewrite this in graphical way

Reply:
what do you mean?
I don't understand :(